recipe/meta.yaml

```diff
@@ -27,10 +27,10 @@ requirements:
 
   run:
     - python
-    - click >=6.6,<8.0.0
+    - click >=6.6
     - cloudpickle >=1.5.0
     - cytoolz >=0.8.2
-    - dask-core >=2021.03.0
+    - dask-core {{ version }}
```

Comment:
Note, we're now pinning `dask-core` version to be the same as the `distributed` version